scripts/question_answering/index.rst

```diff
@@ -0,0 +1,72 @@
+Question Answer
+-------------------
+
+QANet Model
+~~~~~~~~~~~~~~~~~~~~
+
+Reference: A. Yu, D. Dohan et al. "`QANet: Combining Local Convolution with Global Self-Attention for Reading Comprehension <https://arxiv.org/pdf/1804.09541.pdf>`_". ICLR 2018
+
+The dataset used for training and testing the models is `SQuAD <https://arxiv.org/pdf/1606.05250.pdf>`_ v1.0 dataset.
+
+The key features used to reproduce the results for pre-trained models are listed in the following tables.
+
++--------------------------------------+---------------+---------------+--------------------------------------------+--------------------------------------------+--------------+
+|                 Model                | QANet_MXNet_1 | QANet_MXNet_2 |                 QANet_tf_1                 |                 QANet_tf_2                 | origin_paper |
++--------------------------------------+---------------+---------------+--------------------------------------------+--------------------------------------------+--------------+
+|     Embedding     |     Word size    |      300      |      300      |                     300                    |                     300                    |      300     |

```

Comment:
Seems to have some formatting issue here: http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-411/7/model_zoo/question_answering/index.html

Reply:
Fixed.